stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -33,18 +47,278 @@ public void disable(JsonRpcPeer peer, JSONObject params) {
+
+    @JsonProperty
+    public CSSStyle style;
+  }
+
+  private static class CSSStyle {
+    @JsonProperty
+    public String styleSheetId;
+
+    @JsonProperty(required = true)
+    public List<CSSProperty> cssProperties;
+
+    @JsonProperty
+    public List<ShorthandEntry> shorthandEntries;
+
+    @JsonProperty
+    public String cssText;
+
+    @JsonProperty
+    public SourceRange range;
+  }
+
+  private static class ShorthandEntry {
+    @JsonProperty(required = true)
+    public String name;
+
+    @JsonProperty(required = true)
+    public String value;
+
+    @JsonProperty
+    public Boolean imporant;
+  }
+
+  private static class CSSProperty {
+    @JsonProperty(required = true)
+    public String name;
+
+    @JsonProperty(required = true)
+    public String value;
+
+    @JsonProperty
+    public Boolean important;
+
+    @JsonProperty
+    public Boolean implicit;
+
+    @JsonProperty
+    public String text;
+
+    @JsonProperty
+    public Boolean parsedOk;
+
+    @JsonProperty
+    public Boolean disabled;
+
+    @JsonProperty
+    public SourceRange range;
+
+    public CSSProperty(StyleInfo style) {
+      this.name = style.getName();
+      this.value = style.getValue();
+    }
+  }
+
+  private static class PseudoIdMatches {
+    @JsonProperty(required = true)
+    public int pseudoId;
+
+    @JsonProperty(required = true)
+    public List<RuleMatch> matches;
+
+    public PseudoIdMatches() {
+      this.matches = new ArrayList<>();
+    }
+  }
+
+  private static class GetComputedStyleForNodeRequest {
+    @JsonProperty(required = true)
+    public int nodeId;
+  }
+
+  private static class InheritedStyleEntry {
+    @JsonProperty(required = true)
+    public CSSStyle inlineStyle;
+
+    @JsonProperty(required = true)
+    public List<RuleMatch> matchedCSSRules;
+  }
+
+  private static class GetComputedStyleForNodeResult implements JsonRpcResult {
+    @JsonProperty(required = true)
+    public List<CSSComputedStyleProperty> computedStyle;
+
+    public GetComputedStyleForNodeResult(List<StyleInfo> properties) {
+      computedStyle = new ArrayList<>();
+
+      for (int i = 0; i < properties.size(); i++) {
+        computedStyle.add(new CSSComputedStyleProperty(properties.get(i).getName(), properties.get(i).getValue()));
+      }
+    }
+  }
+
+  private static class GetMatchedStylesForNodeRequest implements JsonRpcResult {
+    @JsonProperty(required = true)
+    public int nodeId;
+
+    @JsonProperty
+    public Boolean excludePseudo;
+
+    @JsonProperty
+    public Boolean excludeInherited;
+  }
+
+  private static class GetMatchedStylesForNodeResult implements JsonRpcResult {
+    @JsonProperty
+    public List<RuleMatch> matchedCSSRules;
+
+    @JsonProperty
+    public List<PseudoIdMatches> pseudoElements;
+
+    @JsonProperty
+    public List<InheritedStyleEntry> inherited;
+
+    public GetMatchedStylesForNodeResult(List<StyleInfo> properties) {
+      this.matchedCSSRules = new ArrayList<>();
```

Comment:
In general we haven't been putting any logic into these request/response classes. We should continue that pattern of leaving them as brainless POJOs. In other words, we should do all this preparation outside of this class, not in its constructor.